src/core/renderers/webgl/managers/MaskManager.js

```diff
@@ -52,6 +69,22 @@ MaskManager.prototype.popMask = function (target, maskData)
     {
         this.popSpriteMask(target, maskData);
     }
+    else if (maskData.type === CONST.SHAPES.RECT) {
+        this.popScissorMask(target, maskData);
+    }
+    else if (maskData.graphicsData && maskData.graphicsData.length === 1) {
+        var graphicsData = maskData.graphicsData[0];
+        if (graphicsData.type === CONST.SHAPES.RECT && maskData.rotation === 0 &&
+            maskData.skew.x === 0 && maskData.skew.y === 0)
+        {
+            // The graphics object has no rotation or skew and only a single rectangle in it
```

Comment:
The check above doesn't guarantee there is only a single rectangle in the Graphics object. Just that the first one is a Rectangle.